### DISCUSSION TITLE
Add requirements.pip support

### DISCUSSION THREAD

**Discussion by alchu4n** [aquasecurity/trivy#4191] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/97670155/236613232-df461012-ca10-4253-96da-3fbede2abf37.png)

![image](https://user-images.githubusercontent.com/97670155/236613162-0ce0d917-990f-4ef8-bf54-d3c38ebd9724.png)

**Comment by knqyf263** [comment]
Would you elaborate on that?

**Reply by alchu4n** [reply]
When scanning a python project, it only recognizes the requirements.txt in the project. Some of our research and development often use the requirements.pip file format, which leads to the fact that the corresponding data cannot be obtained at runtime, which can add support for requirements.* style

**Reply by knqyf263** [reply, marked as answer]
Is `requirements.pip` defined somewhere? If you just a different file name of `requirements.txt`, you can use `--file-patterns`.
https://aquasecurity.github.io/trivy/v0.41/docs/configuration/others/#file-patterns

```
--file-patterns "pip:.requirements.*"
```

**Reply by alchu4n** [reply]
thanks
